hack/tools/kapinger/clients/dns.go

```diff
@@ -30,7 +31,7 @@ func (k *KapingerDNSClient) MakeRequests(ctx context.Context) error {
 		case <-ticker.C:
 			go func() {
 				for i := 0; i < k.volume; i++ {
-					domain := "retina.sh"
+					domain := randomString(20) + ".com"
```

Comment:
Please use `.test` instead of `.com` for tests involving DNS per [RFC 2606](https://www.rfc-editor.org/rfc/rfc2606.html)